locales/de/forms.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Yip, done :D json file structured to deliver array of forms per id, in teh code I build a 2d array [pkmn_id, form_int] of all forms available per id, so if event happens some pokemon come with form we can just alter json locale file and we are good to go.